ircbot/plugins/currency.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from urllib.request import urlopen
+
+from ircbot import bot
+
+
+def get_currency(currency_a, currency_b):
+    endpoint = "http://api.fixer.io/latest?base={}".format(str(currency_a).upper())
+
+    response = urlopen(endpoint).read().decode("utf-8")
```

Comment:
Any design decision using urllib here rather than requests (we've been using requests in the rest of botbot, so it would be nice to keep it uniform)